schema/src/lib.rs

```diff
@@ -369,42 +417,6 @@ impl<T: IntoSchema, const L: usize> IntoSchema for [T; L] {
     }
 }
 
-macro_rules! impl_schema_tuple {
-    ($( ( $($id:ident),* ) ),* ) => {$(
-        impl<$($id: IntoSchema),*> IntoSchema for ($($id),*) {
-            fn type_name() -> String {
-                format!("({})", vec![$($id::type_name()),*].join(", "))
-            }
-
-            fn schema(map: &mut MetaMap) {
-                let _ = map.entry(Self::type_name()).or_insert_with(|| {
-                    Metadata::TupleStruct(UnnamedFieldsMeta {
-                        types: vec![$($id::type_name()),*],
-                    })
-                });
-                $(
-                if !map.contains_key(& $id::type_name()) {
-                    $id::schema(map);
-                }
-                )*
-            }
-        }
-    )*};
-}
-
-impl_schema_tuple!(
```

Comment:
I maintain that it's really not a great idea to expose unnamed tuples in schemas. Every tuple can and should be wrapped in a tuple struct at least because it helps human reader understand what the structure represents